sdk/identity/identity/src/msal/nodeFlows/msalOpenBrowser.ts

```diff
@@ -61,12 +61,22 @@ export class MsalOpenBrowser extends MsalNode {
     return this.publicApp!.acquireTokenByCode(request);
   }
 
-  protected doGetToken(
+  private async acquireTokenInteractive(request: msalNode.InteractiveRequest) {
+    if (hasNativeBroker()) {
+      request.windowHandle = nativeBrokerInfo!.options.parentWindowHandle;
+      (request.tokenQueryParameters ??= {})["msal_request_type"] = "consumer_passthrough";
```

Comment:
should we add a check here for `(nativeBrokerInfo!.options.enableMSAPassthrough)` as well?

Reply:
Probably yes